cypress/integration/landing_page_spec.js

```diff
@@ -1,5 +1,83 @@
-describe('Shared Milk user flows', () => {
-  it('Should confirm that true is equal to true', () => {
-    expect(true).to.equal(true)
+describe('Shared Milk landing page user flow', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
```

Comment:
@cagallo remind me, do we want to change this visit link to our heroku link in the future?

Reply:
For development we should stay local, once we're on production we will change to the deployed link!